### DISCUSSION TITLE
Dedicated select all/none/page buttons

### DISCUSSION THREAD

**Discussion by jhf2442** [jonaswinkler/paperless-ng#1125] [Feature Requests]
Currently 2 clicks are needed to select all/none documents : drop down the list, choose the selection mode
Request is to replace the dropdown list with 3 explicit buttons in order to reduce needed number of clicks for a feature that (imho) is used quite often
alternative could be to have a checkbox above the others in the document list, which would switch between select all/none - this is the way phpmyadmin implements the selection

**Comment by shamoon** [comment]
Everything with UI is a balance, yes would reduce clicks to have individual buttons but I’m not sure if “select all” is an every day operation for most people. For me for example it’s pretty rare. Of course without telemetry or other usage data we can never know this for sure but to keep things a little less cluttered we decided to group them into the menu.

Certainly if there is strong support for this we can do it, that’s the cool thing about being able to up-vote feature requests.

**Reply by jhf2442** [reply]
used it a lot during import phase - had lots of documents in my inbox and did batch-processing to set the correspondant, tags etc
now, in everyday usage mode, I have only small set of documents to process at the same time, and the few clocks needed to select all of them if really needed (eg to remove the INBOX tag) is no longer such a burden
